whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapService.java

```diff
@@ -237,7 +237,7 @@ public Response relationSearch(
             @PathParam("key") final String key,
             @QueryParam("status") String status) {
 
-        final RelationType relation = RelationType.fromString(relationType);
+        final RelationType relation = RelationType.fromName(relationType);
```

Comment:
I suggest renaming `fromName` to `fromValue` to be consistent with the naming in the `RelationType` class.

Reply:
explained below, it is fromName because it is using the type.name.